homeassistant/components/nederlandse_spoorwegen/binary_sensor.py

```diff
@@ -0,0 +1,177 @@
+"""Support for Nederlandse Spoorwegen public transport."""
+
+from __future__ import annotations
+
+from collections.abc import Callable
+from dataclasses import dataclass
+import logging
+from typing import Any
+
+import voluptuous as vol
+
+from homeassistant.components.binary_sensor import (
+    PLATFORM_SCHEMA as BINARY_SENSOR_PLATFORM_SCHEMA,
+    BinarySensorEntity,
+    BinarySensorEntityDescription,
+)
+from homeassistant.const import CONF_API_KEY, CONF_NAME, EntityCategory
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import config_validation as cv
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.entity_platform import AddConfigEntryEntitiesCallback
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import (
+    CONF_FROM,
+    CONF_ROUTES,
+    CONF_TIME,
+    CONF_TO,
+    CONF_VIA,
+    DOMAIN,
+    INTEGRATION_TITLE,
+    ROUTE_MODEL,
+)
+from .coordinator import NSConfigEntry, NSDataUpdateCoordinator
+from .utils import (
+    get_arrival_delay,
+    get_coordinator_data_attribute,
+    get_departure_delay,
+    get_going,
+)
+
+_LOGGER = logging.getLogger(__name__)
+
+PARALLEL_UPDATES = 0  # since we use coordinator pattern
+
+ROUTE_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_NAME): cv.string,
+        vol.Required(CONF_FROM): cv.string,
+        vol.Required(CONF_TO): cv.string,
+        vol.Optional(CONF_VIA): cv.string,
+        vol.Optional(CONF_TIME): cv.time,
+    }
```

Comment:
Since you have the same constant on both sensor types, either import one or add a const.py where both import from?